lib/formatters/table.js

```diff
@@ -0,0 +1,134 @@
+"use strict";
```

Comment:
Almost there, can you add the copyright and license lines? (See link above for example)